browser/src/control/jsdialog/Widget.Timefield.js

```diff
@@ -0,0 +1,54 @@
+/* -*- js-indent-level: 8 -*- */
+/*
+ * Copyright the Collabora Online contributors.
+ *
+ * SPDX-License-Identifier: MPL-2.0
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+/*
+ * JSDialog.Timefield - icon view widget
+ *
+ * Example JSON:
+ * {
+ *     id: 'time',
+ *     type: 'time',
+ *     text: '01:01:01'
+ * }
+ */
+
+/* global JSDialog $ */
+
+JSDialog.timeField = function (parentContainer, data, builder) {
+	var container = L.DomUtil.create('span', builder.options.cssClass + ' ui-timefield-container', parentContainer);
+	if (data.id !== undefined) {
+		container.setAttribute('id', data.id);
+	}
+
+	var inputTimeField = L.DomUtil.create('input', builder.options.cssClass + ' ui-timefield', container);
+	inputTimeField.setAttribute('type', 'time');
+	inputTimeField.setAttribute('step', 1); // forces the display of seconds
+	inputTimeField.id = data.id;
```

Comment:
It is incorrect that 2 nodes have the same id (container and inputTimeField).
Please remove container and use only inputTimeField. As I understand there is only one element inside container - please simplify it.